docs/architecture/performance-tests.md

```diff
@@ -0,0 +1,131 @@
+# Performance Tests
+
+### Run Your Own Performance Tests
+The performance of the log ingestion system depends on several custom factors:
+
+- Protocols (UDP/TCP/TLS)
+- Network bandwidth between the source, syslog server, and backend
+- Number of Splunk indexers and third-party SIEMs, including their number and capacity
+- SC4S host's hardware specifications and software configurations
+- The number of syslog sources, the size of their logs, and whether they are well-formed and syslog compliant
+- Customizations
+
+Since actual performance heavily depends on these custom factors, the SC4S team cannot provide general estimates. Therefore, you will need to conduct your own performance tests.
+
+### When to Run Performance Tests
+- To estimate single-instance capacity. The size of the instance must be larger than the absolute anticipated input data peak to prevent data loss.
```

Comment:
### When to run performance tests
- To estimate single-instance capacity. The size of the instance must be larger than the absolute anticipated input data peak to prevent data loss.